packages/app/src/elements/audit.ts

```diff
@@ -0,0 +1,98 @@
+// import { translate as $l } from "@padloc/locale/src/translate";
+import { animateElement } from "../lib/animation";
+import { customElement, state, queryAll } from "lit/decorators.js";
+import { css, html, LitElement } from "lit";
+
+import { shared, mixins } from "../styles";
+import { Routing } from "../mixins/routing";
+
+// TODO: Improve UI
+// TODO: Improve UX
+// TODO: Use translations
+
+@customElement("pl-audit")
+export class Audit extends Routing(LitElement) {
+    @state()
+    private _reusedPasswordCount: number = 0;
+
+    @state()
+    private _weakPasswordCount: number = 0;
+
+    @state()
+    private _compromisedPasswordCount: number = 0;
+
+    @queryAll(".count")
+    private _countElements: HTMLDivElement[];
+
+    static styles = [
+        shared,
+        css`
+            :host {
+                display: block;
+            }
+
+            .count {
+                min-height: 5em;
+            }
+
+            ${mixins.hover(".count")}
+        `,
+    ];
+
+    render() {
+        const { _reusedPasswordCount, _weakPasswordCount, _compromisedPasswordCount } = this;
+        return html`
+            <div class="padded layout">
+                <div class="vertical spacing">
+                    <h3>
+                        Here you can see some counts for passwords you should change, as they might be easier to be
```

Comment:
I like the idea of using filters and routing to the list page, but I think it would be cool to display at least the first 5 of each. You could use the same layout as in https://github.com/padloc/padloc/blob/v4/packages/app/src/elements/org-dashboard.ts, which a "box" for weak, reused, compromised and then up to 5 items in each box (that route to the item directly) and a "View All" button that routes to the list view.